x-pack/plugins/security_solution/server/lib/detection_engine/signals/filters/types.ts

```diff
@@ -44,6 +44,6 @@ export interface CreateFieldAndSetTuplesOptions<T> {
 
 export interface FieldSet {
   field: string;
-  operator: 'excluded' | 'included';
+  operator: 'excluded' | 'included' | 'wildcard_caseless';
```

Comment:
Why was this change needed?

Reply:
So this was needed cause the conditions that depend on the operator values use an `EntriesArray` type from the lists plugin. So without this change [routes/trusted_apps/mapping.ts#L76](https://github.com/elastic/kibana/blob/4eb6c22bcceb6e1400e34ff72bcfd2dc821603ac/x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/mapping.ts#L76) TS complains that the new operator value `wildcard_caseless` never overlaps with `'excluded' | 'included'`.

I could trace this back to [pull/83661](https://github.com/elastic/kibana/pull/83661)

Reply:
Cause TS complains otherwise. Since `FieldSet` is being used in `[create_field_and_set_tuples.ts#L11](https://github.com/FrankHassanabad/kibana/blob/d070f78b957c11f1eeabe8ba86d29a72420d07ca/x-pack/plugins/security_solution/server/lib/detection_engine/signals/filters/create_field_and_set_tuples.ts#L11)` which in turn uses `EntryList` from  'lists' plugin. [entry_list.ts#L20](https://github.com/FrankHassanabad/kibana/blob/d070f78b957c11f1eeabe8ba86d29a72420d07ca/x-pack/plugins/lists/common/schemas/types/entry_list.ts#L20)